ONSdigital/takeon-egestion
# Feature/spp 490/snapshot multi

# Description
Lambda file name change
https://collaborate2.ons.gov.uk/jira/browse/SPP-490

# How to test?
Pass a json message through the spp-es-takeon-egestion-spp490-db-export-input and check the lambda cloud watch log spp-es-takeon-egestion-spp490-export

Test env: spp490-snapshot
unit test
